app/src/main/java/org/fossasia/openevent/app/core/auth/login/LoginFragment.java

```diff
@@ -109,16 +143,4 @@ public void onSuccess(String message) {
     public void showProgress(boolean show) {
         showView(binding.progressBar, show);
     }
-
-    @Override
-    public Lazy<LoginPresenter> getPresenterProvider() {
-        return presenterProvider;
-    }
-
-    @Override
-    public void attachEmails(Set<String> emails) {
```

Comment:
This should have remained

Reply:
So it means that I needn't remove the attachEmail( ) from the interface?

Reply:
It wasn't being directly accessed in MVP, we want to not change view code that much from MVP. Just add observables